dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoTagResource.java

```diff
@@ -109,7 +109,8 @@ public DockerRepoTagResource(ToolDAO toolDAO, TagDAO tagDAO) {
 
                 // If any paths have changed then set dirty bit to true
                 if (!existingTag.getCwlPath().equals(tag.getCwlPath()) || !existingTag.getWdlPath().equals(tag.getWdlPath())
```

Comment:
Kinda wonder if this would be cleaner with https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/EqualsBuilder.html although its not the normal use of it